Kernel/Arch/x86_64/Processor.cpp

```diff
@@ -628,26 +620,29 @@ UNMAP_AFTER_INIT void Processor::early_initialize(u32 cpu)
 
     m_deferred_call_pool.init();
 
-    cpu_setup();
-    gdt_init();
+    self->cpu_setup();
+    self->gdt_init();
 
-    VERIFY(is_initialized());   // sanity check
-    VERIFY(&current() == this); // sanity check
+    VERIFY(is_initialized());
+    VERIFY(&current() == this);
```

Comment:
seems unnecessary to remove the comments here that say //sanity check, esp since they are kept below.